website/www/site/content/en/blog/beamquest.md

```diff
@@ -0,0 +1,35 @@
+---
+title:  "Getting started with Apache Beam: An open source proficiency credential sponsored by Google Cloud"
+date:   2023-06-09 00:00:01 -0800
+categories:
+  - blog
+aliases:
+  - /blog/2023/06/09/beam-quest.html
+authors:
+  - svetaksundhar
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<img class="center-block"
+     src="/images/blog/apch-beam-w_bdg_c_en.png"
+     alt="Quest image">
+
+The [Getting Started with Apache Beam quest](https://www.cloudskillsboost.google/quests/310) is a series of four labs that teach and test proficiency in Apache Beam. Upon completion, you're granted a sharable credential badge!
```

Comment:
do we need these two sentences here?

Reply:
I agree. Let's remove this one.

Reply:
Done. Moved the quest link and removed link from "Google Cloud Badge" (bc its the same link)